docs/UserGuide.md

```diff
@@ -109,24 +109,20 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
-### Locating persons by name: `find`
+### Locating persons by person's information: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds persons based on a category of a person's information, e.g. name, email, phone number, or tags.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find n/NAME|e/EMAIL|p/PHONE|t/TAG`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Only the parameter specified in the command is searched. e.g. `find n/John` will only search for person's name.
+* The search is case-insensitive. e.g `hans` will match `Hans`.
+* Only full words will be matched. e.g. `Alex` will not match `Alex Tan`.
```

Comment:
This line is confusing to me, does it mean `find n/Alex` will not match a person named `Alex Tan`?

Reply:
Yes. From what we discussed previously, I think we agreed to match by exact words? The current behavior of AB3 allows for matching of multiple keywords. Do we want to keep the current behavior of AB3 but extend it to multiple parameters? Or keep to what we have previously discussed?

Reply:
@Dethada @KohGuanZeh It would be better to keep the current behaviour of AB3. I think changing the example to include the command used will clarify the scope of the above-mentioned statements